src/components/Homepage/LandingPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { List } from "./List";
+import { Button } from "./Button";
+import { Description } from "./Description";
+import { Logo } from "./Logo";
+import styled from "@emotion/styled";
+import background from "../../static/svg/backgroundImage.svg";
+
+const BackgroundImage = styled.div`
+  background-image: url(${background});
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+`;
+export const LandingPage = () => (
+  <BackgroundImage>
+    <Logo />
+    <div className="w-5/6 m-auto text-center pt-24 pb-24 lg:w-3/5">
```

Comment:
I would create a string for the classname and use it here and in the second usage on line-35. It's the same classname.